src/app/pages/lesson/lesson.component.html

```diff
@@ -37,7 +37,7 @@
                 <ng-container matColumnDef="deleteLesson">-->
                     <mat-header-cell *matHeaderCellDef>מחיקת שיעור</mat-header-cell>
                     <mat-cell *matCellDef="let row">
-                      <button mat-icon-button color="primary" (click)="deleteLesson(row._id)"><i class="material-icons"> delete </i></button>
+                      <button mat-icon-button color="primary" (click)="deleteLesson(row._id, row.title)"><i class="material-icons"> delete </i></button>
```

Comment:
Why `row.title` is needed to remove lesson?

Reply:
because of the current structure in the database, take a look at the schedule object in the class collection, we keep lessons by title not by _id, so when we need to see if some lesson in connected to a class, we search for it by title, not _id.
I think that we need some refactoring here, but this is outside the scope of this story.